freshmaker/parsers/internal/manual_rebuild.py

```diff
@@ -44,16 +44,21 @@ def parse_post_data(self, data):
             from the UMB message sent from Frontend to Backend.
         """
         msg_id = data.get('msg_id', "manual_rebuild_%s" % (str(time.time())))
-        errata_id = data.get('errata_id')
         dry_run = data.get('dry_run', False)
 
-        errata = Errata()
-        advisory = ErrataAdvisory.from_advisory_id(errata, errata_id)
+        if data.get('errata_id', None):
```

Comment:
Is `errata_id` set if `freshmaker_event_id` is supplied instead? If it's not, then this conditional will not work as intended.

Reply:
I think it's set in `post()` method on *views.py:521*. If `freshmaker_event_id` is supplied then it sets `errata_id` to the search_key of the dependent event.